_reports/saksham.md

```diff
@@ -32,9 +32,6 @@ social:
   - GitLab:
       - username: anctartica
       - link: https://gitlab.com/anctartica
-  - Gitter:
-      - username: anctartica
-      - link: https://gitter.im/anctartica
```

Comment:
i have renamed all my accounts from anctartica to saksham189 so, we need only to update all the links rather than removing.